packages/amplify-e2e-core/src/asciinema-recorder.ts

```diff
@@ -53,6 +54,9 @@ export class Recorder {
     if (this.exitCode !== undefined) {
       throw new Error('Already executed. Please start a new instance');
     }
+    if (process.env.VERBOSE_LOGGING_DO_NOT_USE_IN_CI_OR_YOU_WILL_BE_FIRED || !isCI()) {
```

Comment:
If we're going to get fired by using this in CI, why not add a second another check here that exits w/ code 1 if this is used in a ci environment?
```suggestion
    if (process.env.VERBOSE_LOGGING_DO_NOT_USE_IN_CI_OR_YOU_WILL_BE_FIRED && !isCI()) {
    process.exit(1) // you're fired
    }
    if (process.env.VERBOSE_LOGGING_DO_NOT_USE_IN_CI_OR_YOU_WILL_BE_FIRED || !isCI()) {
```

Reply:
No, I use it :P